core/templates/components/skill-selector/select-skill-modal.component.spec.ts

```diff
@@ -87,14 +86,16 @@ describe('Select Skill Modal', () => {
     componentInstance.categorizedSkills = categorizedSkills;
     componentInstance.skillsInSameTopicCount = skillsInSameTopicCount;
     componentInstance.skillSummaries = skillSummaries;
-    componentInstance.untriagedSkillSummaries = untriagedSkillSummaries;
+    componentInstance.untriagedSkillSummaries =
+    (untriagedSkillSummaries as unknown) as SkillSummaryBackendDict[];
```

Comment:
```suggestion
    componentInstance.untriagedSkillSummaries = (
      untriagedSkillSummaries as unknown as SkillSummaryBackendDict[]);
```

Why is the `unknown` needed?

Reply:
It is typescript rules on how to convert a type into another type.